packages/blocks/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
+
+### New Feature
+
+- Blocks can now be registered with an `allowDefaultStyle` setting, allowing the default style picker to be removed.
```

Comment:
Noting that this doesn't match the documented `defaultStyle` (changed during iterations?)

Reply:
👍🏻 3a140342b